WebInstall/explainer_cross_domain.md

```diff
@@ -0,0 +1,259 @@
+
+```json
+{
+    "name": "Awesome PWA",
+    "display": "standalone",
+    "start_url": "/index.html",
+    "install_sources": [ 
+	    {"origin": "apps.microsoft.com"},
+	    {"origin": "store.app"}
+    ]
+}
+```
+
+This new manifest field will protect the app from being listed in undesirable repositories and give the developer absolute control about where do they want the PWA to be installed from. At best, the developer can allow the PWA to be installed from any site ("`*`"), at its most restrictive, it can only allow installing from the app's same scope. This field is only for the JS API and does not interfere with existing ways of installing PWAs through mechanisms like enterprise policies.
+
+If no `install_sources` are present in the manifest file, the default should be to not allow an app to be installed from cross-origin sites.
+
+#### **Gating capability behind installation**
+A UA may choose to gate the `navigator.install` capability behind installation of the web content. This would serve as an additional trust signal from the user towards enabling the functionality.
+
+**For cross-origin installs, the user gesture, the new origin permission, the new manifest field the final installation confirmation (current default behaviour in the browser before installing an app) and the optional gated capability work together to minimize the risk of origins spamming the user for unrequested installations**, give developers complete flexibility about where their apps will be installed from and provide the user with an implicit (double: one for the user gesture, the other one from the prompt before installing) confirmation before the app gets installed on their device.
+
+## Alternative Solutions
+
+* **HTML anchor tag target install
+`<a href="https://airhorner.com" target="_install">honk</a>`:** An alternate solution to allow installation of web apps is by allowing a new target type of `_install` to the HTML anchor tag. This has the benefit of being able to work in environments that have JS disabled and can also be another entry point for installation. While this is an elegant solution, it limits the amount of information a developer can act upon that the promise does provide, such as if the prompt was shown or if the origin has permissions to install apps.
+
+## Open Questions
+
+* Should we enable a [try-before-you-buy](https://github.com/PEConn/web-install-explainer/blob/main/explainer.md#try-before-you-buy) flow scenario for web install?
+
+* Should we provide feedback to the directory?
```

Comment:
Is this still an open question?  Haven't you addressed this already by both allowing the installation origin to be aware of the success, as well as the ability to call `getInstalledApps()`?